scripts/check_setup.sh

```diff
@@ -33,46 +36,50 @@ else
 fi
 
 # Output required dependencies
-if [ $HAS_REQUIRED = false ] ; then
-    echo "\n\n${COLOR_RED}${BOLD}The following required required dependencies are missing:${RESET_STYLE}\n"
+if [[ $HAS_REQUIRED = false ]] ; then
+    printf "\n\n $COLOR_RED $BOLD The following required required dependencies are missing: $RESET_STYLE \n"
```

Comment:
Nit: I'm getting a lint warning on all these `printf` calls to avoid using the variables directly in the string and use a formatter string instead like:

```bash
printf "\n\n %s %s The following required required dependencies are missing: %s \n" "$COLOR_GREEN" "$BOLD" "$RESET_STYLE"
```

Does that make more sense or do we want to leave it?